pynest/nest/lib/hl_api_parallel_computing.py

```diff
@@ -86,3 +86,19 @@ def SetAcceptableLatency(port, latency):
 
     sps(latency)
     sr("/%s exch SetAcceptableLatency" % port)
+
+
+@check_stack
+def SetMaxBuffered(port, size):
+    """Set the maximum buffer size for a MUSIC port.
+
+    Parameters
+    ----------
+    port : object
+        MUSIC port to set buffer size for
+    size : int
+        Buffer size
+    """
+
+    sps(size)
+    sr("/%s exch SetMaxBuffered" % port)
```

Comment:
1. Couldn't this be 
```
sps(SLILiteral(port))
sps(size)
sr("SetMaxBuffered")
```
That would be clearer, if that works correctly and isn't too much work
2. Otherwise, I'd use >= python2.7 calls:
```
sr("/{} exch SetMaxBuffered".format(port))
```
or even (to keep it simple)
```
sr("/{} {} SetMaxBuffered".format(port, size)
```
with no `sps(size)`